NOTICE

```diff
@@ -1,4 +1,4 @@
-reports-job
-Copyright 2017 Fintech Open Source Foundation
+reports-job - FINOS
+Copyright 2019 Fintech Open Source Foundation
```

Comment:
this change is not needed, tho I'll merge it anyway. The year mentioned in the copyright must be the year of the initial contribution, not the current year.